build_scripts/build_linux.sh

```
@@ -0,0 +1,104 @@
+        echo "        Option 1: sudo apt install python3-pip && python3 -m pip install --user 'pyinstaller>=6.0.0'"
+        echo "        Option 2: sudo apt install pyinstaller"
+        exit 1
+    fi
+else
+    echo "      PyInstaller found"
+fi
+
+# Clean previous builds
+echo "[3/5] Cleaning previous builds..."
+cd ..
+rm -rf "${BUILD_PATH}/" "${DIST_PATH}/"
```

Shellcheck (0.11.0)

[warning] Use "${var:?}" to ensure this never expands to / .

(SC2115)